powerauth-nextstep/src/main/java/io/getlime/security/powerauth/app/nextstep/configuration/SecurityConfig.java

```diff
@@ -31,19 +39,57 @@
  */
 @Configuration
 @EnableWebSecurity
+@Slf4j
 public class SecurityConfig {
 
+    @Value("${powerauth.nextstep.security.auth.type:NONE}")
+    private AuthType authType;
+
     /**
      * Configures HTTP security.
+     *
      * @param http HTTP security.
      * @throws Exception Thrown when configuration fails.
      */
     @Bean
     public SecurityFilterChain filterChain(final HttpSecurity http) throws Exception {
+        if (authType == AuthType.OIDC) {
+            logger.info("Initializing OIDC authentication.");
+            http.authorizeHttpRequests(authorize -> authorize
+                            .requestMatchers(
+                                    new AntPathRequestMatcher("/login/oauth2/**"),
```

Comment:
Only 

- actuator
- status 
- and the  /login/ouath2
should be permitted.


Swagger console and docs should be accessible only for authorized user (same as in PAC)